glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: 7c7d875c128e2f5ee2218b87f3ec2b6fb335279d1cda2657c6cc4eb92a2ecdb9
-updated: 2018-01-17T11:45:33.306560761-08:00
+updated: 2018-02-04T15:09:19.499281-08:00
```

Comment:
why the glide changes?

Reply:
ahh my first yarpc change and did glide up locally to install the dep. will remove.

Reply:
I tried reverting but it was causing a `ineffective assignment to field` on linter. I put it back.

Reply:
We shouldn't need glide changes for this diff, so we should remove it. I don't know where that error is coming from, but if glide update is needed, it should be done separately.